runtime-config-linux.md

```diff
@@ -250,6 +250,16 @@ The actions and operators are strings that match the definitions in seccomp.h fr
    }
 ```
 
+## Masked Paths
+
+Masked paths allow hiding paths in the container by bind mounting `/dev/null` over them to prevent access by the container process.
```

Comment:
On Fri, Sep 25, 2015 at 11:17:19AM -0700, Vincent Batts wrote:
> > +Masked paths allow hiding paths in the container by bind mounting
> > `/dev/null` over them to prevent access by the container process.
> 
> perhaps that mount approach is an implementation approach, but not
> the requirement?  If some … cow fs … feature allowed the filepath to
> just not exist…

What would this look like?  Looking for similar tech besides “bind
over it”, there's the device cgroups [1].  But there the access is
bound to major/minor, and not by path.  That's not a big deal though,
I could see something like that bound by path.  But the cgroup
approach gives errors like:

  # cgcreate -g devices:/device-group
  # cgset -r devices.deny='c 1:3 mrw' device-group
  # cgexec -g devices:/device-group cat /dev/null
  cat: /dev/null: Operation not permitted
  # cgdelete -g devices:/device-group

and not just an empty read.  So I'd be cautious about saying “don't
worry about how it's implemented, just use this to block access”.

[1]: https://www.kernel.org/doc/Documentation/cgroups/devices.txt